### ISSUE TITLE
Editor Accessibility regression: no labels for some tabs

### ISSUE THREAD

**Issue by Nuno69** [godotengine/godot#114184]
### Tested versions

4.6 Beta 1

### System information

Godot v4.5.1.stable - macOS Tahoe (26.2.0) - Multi-window, 1 monitor - OpenGL 3 (Compatibility) - Apple M4 - Apple M4 (10 threads) - 24.00 GiB memory

### Issue description

In Godot Engine 4.6 beta same tabs have no labels for screen reader. Namely "Debugger", "Audio", "Animation", "Shader editor". In the latest stable (V 4.5.1) they have labels.

### Steps to reproduce

1. Open a project
2. 2. Use the tab key to navigate around the window.
3. 3. You will find some unlabelled tabs.
The expected result is that those label exist and are read by a screen reader.

### Minimal reproduction project (MRP)

No need

**Comment by Nuno69** [comment]
Hi,
thanks for your reply. Is that easy to fix?

**Comment by bruvzg** [comment]
> Is that easy to fix?

Probably, I'll check it on this weekend.
